pygame/rwobject.py

```diff
@@ -1,10 +1,62 @@
 """ The pygame rwobject module for IO using SDL_RWops """
 
-from pygame._sdl import sdl
+from pygame._sdl import ffi, sdl
 from pygame._error import SDLError
 from pygame.compat import bytes_, filesystem_encode, unicode_
 
 
+# Callback helpers for rwops_from_file
+@ffi.callback("int (SDL_RWops* context, int offset, int whence)")
+def obj_seek(context, offset, whence):
+    fileobj = ffi.from_handle(context.hidden.unknown.data1)
+    if not hasattr(fileobj, 'tell') or not hasattr(fileobj, 'seek'):
+        return -1
```

Comment:
Should these checks be just before `seek` and `tell` are actually used so we don't end up calling a method that doesn't exist?